core/src/main/java/com/google/googlejavaformat/java/UsageException.java

```diff
@@ -52,6 +52,10 @@
     "    Character offset to format (0-based; default is all).",
     "  --help, -help, -h",
     "    Print this usage statement",
+    "  --validate, -validate",
+    "    Validate files format only. If files need format print its path to stdout.",
+    "  --set-exit-if-changed, -set-exit-if-changed",
+    "    Validate files only, do not write anything. Error code 0 means everything's formatted.",
```

Comment:
I prefer:

```
--dry-run -n
  Prints the paths of the files whose contents would change if the formatter were run normally.
--set-exit-if-changed
  Return exit code 1 if there are any formatting changes
```

validation isn't necessarily the only use of these flags.

Reply:
Ok.

Reply:
Support `-e` as a short-cut for `--set-exit-if-changed`?

Reply:
I think it's fine to skip the short form for this one.

Reply:
Ok.